public/app/features/plugins/admin/components/PluginDetailsPanel.tsx

```diff
@@ -16,73 +29,191 @@ type Props = {
+                  </Text>
+                  <Tooltip
+                    content={
+                      <Trans i18nKey="plugins.details.labels.customLinksTooltip">
+                        These links are provided by the plugin developer to offer additional, developer-specific
+                        resources and information
+                      </Trans>
+                    }
+                    placement="right-end"
+                  >
+                    <Icon name="info-circle" size="xs" />
+                  </Tooltip>
+                </Stack>
+              }
+            >
+              <Stack direction="column" gap={2}>
+                {customLinks.map((link, index) => (
+                  <TextLink key={index} href={link.url} external>
+                    {link.name}
+                  </TextLink>
+                ))}
+              </Stack>
+            </CollapsableSection>
+          </Box>
+        )}
+        {!plugin?.isCore && (
+          <Box padding={2} borderColor="medium" borderStyle="solid">
+            <CollapsableSection
+              headerDataTestId="reportConcern"
+              isOpen={false}
+              label={
+                <Stack direction="row" justifyContent="space-between" alignItems="center">
+                  <Text color="secondary" variant="body">
+                    <Trans i18nKey="plugins.details.labels.reportAbuse">Report a concern </Trans>
+                  </Text>
+                  <Tooltip
+                    content={
+                      <Trans i18nKey="plugins.details.labels.reportAbuseTooltip">
+                        Report issues related to malicious or harmful plugins directly to Grafana Labs.
+                      </Trans>
+                    }
+                    placement="right-end"
+                  >
+                    <Icon name="info-circle" size="xs" />
+                  </Tooltip>
+                </Stack>
+              }
+            >
+              <Stack direction="column">
+                <Button variant="secondary" fill="solid" icon="bell" onClick={() => setReportAbuseModalOpen(true)}>
+                  <Trans i18nKey="plugins.details.labels.contactGrafanaLabs">Contact Grafana Labs</Trans>
+                </Button>
+              </Stack>
+            </CollapsableSection>
+          </Box>
+        )}
+      </Stack>
+      {reportAbuseModalOpen && (
+        <Modal
+          title={<Trans i18nKey="plugins.details.modal.title">Report a plugin concern</Trans>}
+          isOpen
+          onDismiss={() => setReportAbuseModalOpen(false)}
+        >
+          <Stack direction="column" gap={2}>
+            <Text>
+              <Trans i18nKey="plugins.details.modal.description">
+                This feature is for reporting malicious or harmful behaviour within plugins. For plugin concerns, email
+                us at: <TextLink href="mailto:integrations@grafana.com">integrations@grafana.com</TextLink>.{' '}
```

Comment:
```suggestion
                us at: <TextLink href="mailto:integrations+report-plugin@grafana.com">integrations@grafana.com</TextLink>.{' '}
```
might be easier to separate those requests out this way.